adapters/Maio/MaioAdapter/GADMediationAdapterMaio.m

```diff
@@ -59,8 +85,23 @@ + (void)setUpWithConfiguration:(GADMediationServerConfiguration *)configuration
   }];
 }
 
++ (void)setUpCaseNotExistsAnyWithCompletionHandler: (nonnull GADMediationAdapterSetUpCompletionBlock) completionHandler {
+  NSError *error = GADMAdapterMaioErrorWithCodeAndDescription(
+          GADMAdapterMaioErrorInvalidServerParameters,
+          @"maio mediation configurations did not contain a valid media ID.");
+  completionHandler(error);
+}
+
++ (void)setupCaseExistsPublisherIDs: (nonnull NSSet *)publisherIDs completionHandler: (nonnull GADMediationAdapterSetUpCompletionBlock) completionHandler {
+  completionHandler(nil);
```

Comment:
Does open-bidding not require the maio SDK to be initialized? If so, can we add a comment here to explicitly say so for reference ?